frontend/src/metabase/query_builder/components/notebook/steps/JoinStep/JoinStep.unit.spec.tsx

```diff
@@ -278,6 +284,39 @@ describe("Notebook Editor > Join Step", () => {
     ).not.toBeInTheDocument();
   });
 
+  it("questions from another database should not appear in recents", async () => {
+    setup(undefined, {
+      readOnly: false,
+      recentItems: [
+        createMockRecentCollectionItem({
+          model: "card",
```

Comment:
:x: It's a model. This change will make the test fail.



```suggestion
          model: "dataset",
```